tests/integration/inventory.networking

```diff
@@ -3,9 +3,14 @@ lh-dmz1-pod1-mso-v224 ansible_host=173.36.219.29 ansible_connection=local mso_ho
 lh-dmz1-pod1-mso-v311 ansible_host=173.36.219.11 ansible_connection=local mso_hostname=173.36.219.11
 lh-dmz1-pod1-ndo-v371 ansible_host=173.36.219.30 ansible_connection=ansible.netcommon.httpapi ansible_network_os=cisco.nd.nd mso_hostname=173.36.219.30
 
-[mso:vars]
+[ndo]
+lh-dmz1-pod1-ndo-v400 ansible_host=173.36.219.32 ansible_connection=ansible.netcommon.httpapi ansible_network_os=cisco.nd.nd ndo_hostname=173.36.219.32
+
+[all:vars]
```

Comment:
doesn't this affect the usage of other modules?

Reply:
I have tested on other module for it to still work, this is also why I renamed mso:vars to all:vars. Due to the prepend change of the new module I needed to make a change to this to get it working. We can discuss alternative methods if you have a better approach.